src/components/Card/Card.tsx

```diff
@@ -1,16 +1,47 @@
 import React from 'react';
 import { Post } from '../App/App';
+import 'dayjs';
 import './Card.css';
+import dayjs from 'dayjs';
 
 interface CardProps {
   post: Post;
 }
 
 export const Card: React.FC<CardProps> = ({ post }) => {
+  const formatDateCreated = () => {
+    let now = dayjs();
+    let createdAt = dayjs(post.createdAt);
+    let diff = now.diff(createdAt, 'hours');
+
+    if (diff >= 24) {
+      let time = dayjs(post.createdAt).format('h:mm a');
+      let date = dayjs(post.createdAt).format('MMMM D, YYYY');
+      return `at ${time} on ${date}`;
+    } else {
+      return `${diff}hr ago`;
+    }
+  };
+
   return (
     <button className='card'>
```

Comment:
We can avoid the problem of the entire card being a button by having the upvote or downvote just outside, in a different div? It's a bit div soupy, but it sort of looks like reddit does that.

Reply:
Oh yeah love it